fb_instant_articles.module

```diff
@@ -8,28 +8,14 @@
  * Implements hook_menu().
  */
 function fb_instant_articles_menu() {
-  // RSS Endpoint.
-  $items['fbinstant.rss'] = array(
-    'page callback' => 'fb_instant_articles_rss_page',
-    'file' => 'includes/rss.inc',
-    'access arguments' => array('access content'),
-  );
-
   // Top level configuration page.
   $items['admin/config/services/fb-instant-articles'] = array(
-    'title' => 'Facebook Instant Articles Settings',
-    'description' => 'Facebook Instant Articles',
-    'page callback' => 'drupal_get_form',
-    'page arguments' => array('fb_instant_articles_admin_settings_form'),
-    'access arguments' => array('administer fb_instant_articles'),
-    'file' => 'includes/admin.inc',
-  );
-
-  // Configure Facebook Instant Article settings.
-  $items['admin/config/services/fb-instant-articles/settings'] = array(
-    'title' => 'Settings',
-    'type' => MENU_DEFAULT_LOCAL_TASK,
-    'weight' => -1,
+    'title' => 'Facebook Instant Articles',
+    'description' => 'Settings related to Facebook Instant Articles.',
+    'page callback' => 'system_admin_menu_block_page',
+    'access arguments' => array('access administration pages'),
+    'file' => 'system.admin.inc',
+    'file path' => drupal_get_path('module', 'system'),
```

Comment:
Shouldn't this be:

```php
// ...
'page callback' => 'drupal_get_form',
'page arguments' => array('fb_instant_articles_admin_settings_form'),
'access arguments' => array('administer fb_instant_articles'),
'file' => 'includes/admin.inc',
// ...
```

As it is there is nothing rendered at `/admin/config/services/fb-instant-articles`.